mm2src/coins/nft.rs

```diff
@@ -306,13 +314,71 @@ pub async fn update_nft(ctx: MmArc, req: UpdateNftReq) -> MmResult<(), UpdateNft
             &req.url_antispam,
         )
         .await?;
+        update_nft_global_in_coins_ctx(&ctx, &storage, *chain, nft_global).await?;
         update_transfers_with_empty_meta(&storage, chain, &req.url, &req.url_antispam).await?;
         update_spam(&storage, *chain, &req.url_antispam).await?;
         update_phishing(&storage, chain, &req.url_antispam).await?;
     }
     Ok(())
 }
 
+/// Updates the global NFT information in the coins context.
+///
+/// This function uses the up to date NFT list for a given chain and updates the
+/// corresponding global NFT information in the coins context.
+///
+/// # Arguments
+///
+/// * `ctx`: Shared global context containing the coins context.
+/// * `storage`: Storage interface for NFT list operations.
+/// * `chain`: The blockchain to retrieve NFT information for.
+/// * `nft_global`: Currently the EthCoin object to update with the latest NFT information.
```

Comment:
Here too. For primitive types, clean argument names should help enough. As for the more complicated types, we should already document them on their own.

Reply:
Done